ssucli/ssucli.cpp

```diff
@@ -321,6 +325,11 @@ void SsuCli::optModifyRepo(enum Actions action, QStringList opt)
             return;
         }
 
+        if (action == Add && sysRepos.contains(repo)) {
+            qerr << "Cannot override system repository '" << opt.at(2) << "'" << endl;
```

Comment:
Worth printing something similar when trying to remove system repository?

Reply:
It would have to be the other list of default system repos, as there are others like tools and qa which aren't present by default and we can add or remove at will. I'm not sure if this is the same level of accidental breakage that we're trying to catch here though. Will have a look how easy it is.

Reply:
Now it won't let you disable or remove a default repo, on both the ssud side and in ssucli. Not sure if the disabling is overkill, but it will probably stop someone somewhere from breaking their upgrade.